Source/SelfService/Web/microservice/microserviceDetails/configurationFilesSection/filesSection/filesSection.tsx

```diff
@@ -42,16 +42,17 @@ export const FilesSection = ({ applicationId, environment, microserviceName, mic
     const [validateFileDialogIsOpen, setValidateFileDialogIsOpen] = useState({ isOpen: false, file: [] as File[] });
 
     useEffect(() => {
-        fetchAndUpdateConfigFileNamesList()
-            .catch(console.error);
+        fetchAndUpdateConfigFileNamesList();
     }, []);
 
     const fetchAndUpdateConfigFileNamesList = async (): Promise<void> => {
-        const result = await getConfigFilesNamesList(applicationId, environment, microserviceId);
+        const result = await getConfigFilesNamesList(applicationId, environment, microserviceId)
```

Comment:
Does this work? when you await the `getConfigFilesNamesList` method, can you then treat it as a promise?

Either use promise chaining the entire way, or wrap the call in a try catch like this: 
```
try {
    const result = await getConfigFilesNamesList(applicationId, environment, microserviceId)
    createDataTableObj(result.data ?? [] );
} catch {
     enqueueSnackbar('Could not fetch config files.', { variant: 'error' });
}
```